plugin-examples/pom.xml

```diff
@@ -1,205 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>com.hotels.styx</groupId>
+    <artifactId>styx-parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
 
   <modelVersion>4.0.0</modelVersion>
-  <packaging>jar</packaging>
 
-  <groupId>com.hotels.styx</groupId>
   <artifactId>plugin-examples</artifactId>
   <name>Styx proxy plugin examples</name>
-  <version>0.0.1-SNAPSHOT</version>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
-
-    <!-- Styx Version -->
-    <styx.version>0.9-SNAPSHOT</styx.version>
-    <rxjava.version>1.1.6</rxjava.version>
-    <jackson.version>2.9.9</jackson.version>
-    <!--Testing-->
-    <testng.version>6.8</testng.version>
-
-    <!-- apache plugin versions and configurations, please sort alphabetically -->
-    <maven-compiler-plugin.version>3.0</maven-compiler-plugin.version>
-    <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
-    <maven-resources-plugin.version>2.6</maven-resources-plugin.version>
-    <maven-surefire-plugin.version>2.16</maven-surefire-plugin.version>
+    <main.basedir>${project.parent.basedir}</main.basedir>
   </properties>
 
-
   <dependencies>
     <dependency>
       <groupId>com.hotels.styx</groupId>
       <artifactId>styx-api</artifactId>
-      <version>${styx.version}</version>
-      <!-- The dependency is explicitlu needed only during compile time, styx is already available in the runtime-->
-      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>io.reactivex</groupId>
       <artifactId>rxjava</artifactId>
-      <version>${rxjava.version}</version>
-      <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
As mentioned, all these changes in the dependencies mean that the pom file cannot be just copied to a new plugin. At some point we should make sure our BOM file does not expose all the dependencies of Styx and then have plugins import it...